### PR TITLE
[SPARK-7927] whitespace fixes for SQL core.

### PR DESCRIPTION
**Pull request by rxin**
So we can enable a whitespace enforcement rule in the style checker to save code review time.

### CODE REVIEW

**Comment by vanzin** [comment]
So, has something changed in the community's previous assessment that making these large, style-only changes are not worth it (because they can make merges difficult, and also make history / git blame harder to figure out)?

I still have the code for SPARK-3873 which should still work - 99% of the patch is just fixing the existing source files.

File: sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala [hunk 1]
```diff
@@ -52,6 +52,7 @@ private[sql] object JDBCRDD extends Logging {
       scale: Int,
       signed: Boolean): DataType = {
     val answer = sqlType match {
+      // scalastyle:off
```
Comment (liancheng) [inline sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala]: Why do we want to turn off the check here?

Comment (rxin) [inline sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala]: just too many things I didn't bother changing.

Comment (liancheng) [inline sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala]: I see... Regex replacing rocks here :)

**Pull request closed without merging**